2021-02-21 02:42:14,214 INFO    MainThread:295938 [wandb_setup.py:_flush():70] setting env: {}
2021-02-21 02:42:14,214 INFO    MainThread:295938 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-02-21 02:42:14,214 INFO    MainThread:295938 [wandb_setup.py:_flush():70] setting login settings: {}
2021-02-21 02:42:14,214 INFO    MainThread:295938 [wandb_init.py:_log_setup():319] Logging user logs to /home/astro/Documents/UvA/Block 4 - NLP2/Multimodal NLP/Multimodal-NLP/wandb/run-20210221_024214-2l5864jg/logs/debug.log
2021-02-21 02:42:14,214 INFO    MainThread:295938 [wandb_init.py:_log_setup():320] Logging internal logs to /home/astro/Documents/UvA/Block 4 - NLP2/Multimodal NLP/Multimodal-NLP/wandb/run-20210221_024214-2l5864jg/logs/debug-internal.log
2021-02-21 02:42:14,215 INFO    MainThread:295938 [wandb_init.py:init():351] calling init triggers
2021-02-21 02:42:14,215 INFO    MainThread:295938 [wandb_init.py:init():358] wandb.init called with sweep_config: {}
config: {}
2021-02-21 02:42:14,215 INFO    MainThread:295938 [wandb_init.py:init():404] starting backend
2021-02-21 02:42:14,223 INFO    MainThread:295938 [backend.py:ensure_launched():81] starting backend process...
2021-02-21 02:42:14,229 INFO    MainThread:295938 [backend.py:ensure_launched():86] started backend process with pid: 295996
2021-02-21 02:42:14,230 INFO    MainThread:295938 [wandb_init.py:init():413] backend started and connected
2021-02-21 02:42:14,231 INFO    MainThread:295938 [wandb_init.py:init():436] updated telemetry
2021-02-21 02:42:14,231 INFO    MainThread:295938 [wandb_init.py:init():459] communicating current version
2021-02-21 02:42:14,586 INFO    MainThread:295938 [wandb_init.py:init():464] got version response 
2021-02-21 02:42:14,586 INFO    MainThread:295938 [wandb_init.py:init():472] communicating run to backend with 30 second timeout
2021-02-21 02:42:14,769 INFO    MainThread:295938 [wandb_init.py:init():495] starting run threads in backend
2021-02-21 02:42:15,528 INFO    MainThread:295938 [wandb_run.py:_console_start():1411] atexit reg
2021-02-21 02:42:15,529 INFO    MainThread:295938 [wandb_run.py:_redirect():1274] redirect: SettingsConsole.REDIRECT
2021-02-21 02:42:15,529 INFO    MainThread:295938 [wandb_run.py:_redirect():1279] Redirecting console.
2021-02-21 02:42:15,529 INFO    MainThread:295938 [redirect.py:install():213] install start
2021-02-21 02:42:15,529 INFO    MainThread:295938 [redirect.py:install():228] install stop
2021-02-21 02:42:15,529 INFO    MainThread:295938 [redirect.py:install():213] install start
2021-02-21 02:42:15,529 INFO    MainThread:295938 [redirect.py:install():228] install stop
2021-02-21 02:42:15,530 INFO    MainThread:295938 [wandb_run.py:_redirect():1325] Redirects installed.
2021-02-21 02:42:15,530 INFO    MainThread:295938 [wandb_init.py:init():518] run started, returning control to user process
2021-02-21 02:42:15,533 INFO    MainThread:295938 [wandb_run.py:_atexit_cleanup():1381] got exitcode: 2
2021-02-21 02:42:15,533 INFO    MainThread:295938 [wandb_run.py:_restore():1353] restore
2021-02-21 02:42:15,533 INFO    MainThread:295938 [redirect.py:uninstall():232] uninstall start
2021-02-21 02:42:15,534 INFO    MainThread:295938 [redirect.py:_stop():287] _stop: stdout
2021-02-21 02:42:15,534 INFO    stdout    :295938 [redirect.py:_pipe_relay():129] relay done saw last write: stdout
2021-02-21 02:42:15,534 INFO    MainThread:295938 [redirect.py:_stop():293] _stop closed: stdout
2021-02-21 02:42:15,534 INFO    stdout    :295938 [redirect.py:_pipe_relay():145] relay done done: stdout
2021-02-21 02:42:15,534 INFO    MainThread:295938 [redirect.py:_stop():299] _stop joined: stdout
2021-02-21 02:42:15,534 INFO    MainThread:295938 [redirect.py:_stop():301] _stop rd closed: stdout
2021-02-21 02:42:15,534 INFO    MainThread:295938 [redirect.py:uninstall():236] uninstall done
2021-02-21 02:42:15,534 INFO    MainThread:295938 [redirect.py:uninstall():232] uninstall start
2021-02-21 02:42:15,534 INFO    MainThread:295938 [redirect.py:_stop():287] _stop: stderr
2021-02-21 02:42:15,535 INFO    stderr    :295938 [redirect.py:_pipe_relay():129] relay done saw last write: stderr
2021-02-21 02:42:15,535 INFO    MainThread:295938 [redirect.py:_stop():293] _stop closed: stderr
2021-02-21 02:42:15,535 INFO    stderr    :295938 [redirect.py:_pipe_relay():145] relay done done: stderr
2021-02-21 02:42:15,535 INFO    MainThread:295938 [redirect.py:_stop():299] _stop joined: stderr
2021-02-21 02:42:15,535 INFO    MainThread:295938 [redirect.py:_stop():301] _stop rd closed: stderr
2021-02-21 02:42:15,535 INFO    MainThread:295938 [redirect.py:uninstall():236] uninstall done
2021-02-21 02:42:15,842 INFO    MainThread:295938 [wandb_run.py:_wait_for_finish():1504] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 1549
  total_bytes: 1549
}

2021-02-21 02:42:17,845 INFO    MainThread:295938 [wandb_run.py:_wait_for_finish():1504] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 6410
  total_bytes: 6410
}

2021-02-21 02:42:19,046 INFO    MainThread:295938 [wandb_run.py:_show_files():1726] logging synced files
